src/Cake.Common/Tools/SignTool/SignToolResolver.cs

```diff
@@ -67,7 +67,9 @@ private FilePath GetFromDisc()
             if (_environment.Platform.Is64Bit)
             {
                 // 64-bit specific paths.
-                files.Add(programFilesPath.Combine(@"Windows Kits\10\bin\x64").CombineWithFilePath("signtool.exe"));
+                files.Add(programFilesPath.Combine(@"Windows Kits10\bin\x64").CombineWithFilePath("signtool.exe"));
```

Comment:
Was this intentional? Is the folder really named `Windows Kits10`? If yes then same thing should be done on 32-bit side, right?